ILRepack/Steps/AttributesRepackStep.cs

```diff
@@ -88,7 +90,7 @@ void CleanupAttributes()
             {
                 String name = (string)ca.ConstructorArguments[0].Value;
                 int idx;
-                if ((idx = name.IndexOf(", PublicKey=", StringComparison.Ordinal)) != -1)
+                if ((idx = name.IndexOfRegex(new Regex(", ?PublicKey="))) != -1)
```

Comment:
Isn't it fine to take the name from 0 to first occurrence  + trim? Then we don't care about the publickey part at all